molecule/default/molecule.yml

```diff
@@ -6,14 +6,37 @@ driver:
 lint:
   name: yamllint
 platforms:
-  - name: debian9
-    image: debian:stretch-slim
-  - name: debian10
-    image: debian:buster-slim
-  - name: fedora29
-    image: fedora:29
-  # - name: fedora30
-  #   image: fedora:30
+  - name: debian9_systemd
+    image: geerlingguy/docker-debian9-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: debian10_systemd
+    image: geerlingguy/docker-debian10-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  - name: fedora29_systemd
+    image: geerlingguy/docker-fedora29-ansible:latest
+    privileged: yes
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /lib/systemd/systemd
+    pre_build_image: yes
+  # Ansible currently insists on installing python2-dnf, which does
+  # not exist in Fedora 30.  Until that is resolved, we can't use
+  # Fedora 30.
+  # - name: fedora29_systemd
```

Comment:
I think this should still reflect a Fedora 30 entry in the name and image even if it's not currently in use.

Reply:
Fixed in dea41ec.  You have sharp eyes!